packages/contracts/contracts/libs/v0.8.x/minter-libs/DAExpLib.sol

```diff
@@ -36,7 +36,7 @@ library DAExpLib {
         // @dev max uint40 ~= 1.1e12 sec ~= 34 thousand years
         uint40 timestampStart;
         uint40 priceDecayHalfLifeSeconds;
-        // @dev max uint88 ~= 3e26 Wei = ~300 million ETH, which is well above
+        // @dev max uint88 ~= 3e26 Wei = ~30 million ETH, which is well above
```

Comment:
wasn't 300 correct here? (3e26/10^18)

Reply:
oops, fixed in [4a94e1d](https://github.com/ArtBlocks/artblocks-contracts/pull/1091/commits/4a94e1d7da92db22f47d666e2e71340e6a270d6d)